install/tools/ipa-compat-manage

```diff
@@ -100,7 +100,10 @@ def main():
         if dirman_password is None:
             sys.exit("Directory Manager password required")
 
-    api.bootstrap(context='cli', in_server=True, debug=options.debug)
+    api.bootstrap(context='cli',
+                  in_server=True,
+                  debug=options.debug,
+                  confdir=paths.ETC_IPA)
```

Comment:
Yes, there is - we will have to test this and make sure we don't break it, which is not worth doing for a "feature" nobody wants and will ever use.